aten/src/ATen/native/native_functions.yaml

```diff
@@ -9753,20 +9753,20 @@
 
 - func: inner.out(Tensor self, Tensor other, *, Tensor(a!) out) -> Tensor(a!)
 
-# torch.outer, alias for torch.ger
 - func: outer(Tensor self, Tensor vec2) -> Tensor
   variants: function, method
+  dispatch:
+    CompositeExplicitAutograd: outer
```

Comment:
Please fix  the comment in https://github.com/pytorch/pytorch/blob/c769300301e8bc775e54cdd114e2beaa11b1f893/torch/overrides.py#L463 as well

Reply:
(I found this by going down the list in https://github.com/pytorch/pytorch/blob/d7ef9b73fbc971c241bc3a10be4dc368d5786507/aten/src/ATen/native/native_functions.yaml#L220)

Reply:
Ho I didn't knew there were actual instructions for this! Good catch!